File: 67cf5bc8-9d6f-4d09-b351-4d2bb659edee/bot.py
import discord

API_KEY = "ODYzNTgzNjc2MjM4OTIxODAx.YOpA-Q.WiwIuvdGW6TIDWrEr9XcPvynwH4"

bot = discord.Client()

@bot.event
def on_ready():
  print("Connected to: {0}".format(bot.user))

@bot.event
def on_message(message):

  if message.author == bot.user:
    return
  
  if message.content.startswith('$test_string'):
    await message.channel.send("Recieved")
  
  print(message.content)


bot.run(API_KEY)

File: 2dad7506-d667-43b1-bb26-ee19d8712df5/comandos/bot.js
const { Client, MessageEmbed, DiscordAPIError, PermissionOverwrites  } = require("discord.js");
const client = new Client();
 
client.on("message", (message) =>{
    if(message.content.startsWith("!ayuda")){
        message.channel.send("Ayuda de prueba!")
    }
});
 
client.login("OTU0MTMyODM3MTczNDM2NDU3.YjOrcw.PxwxgV0a9kPBSftiNlwmWWwjXok")